floss/main.py

```diff
@@ -466,7 +465,7 @@ def get_signatures(sigs_path):
     return paths
 
 
-def write(results: ResultDocument, json_: bool, verbose: Verbosity, quiet: bool, outfile: Optional[str]):
+def write(results: ResultDocument, json_: bool, verbose: Verbosity, quiet: bool, outfile: Optional[str], console):
```

Comment:
i wonder if we should remove the `--outfile` cli parameter and rely on people redirecting output like `floss > output.json`. there's getting to be a lot of arguments to this function.

Reply:
we should ensure that when `--color` is not explicitly provided and the output is redirect that color is automatically disabled. i think `rich` does this, but please confirm.